infra/tf/better_uptime/main.tf

```diff
@@ -0,0 +1,53 @@
+provider "betteruptime" {
+	api_token = module.betteruptime_secrets.values["better_uptime/token"]
+}
+
+module "betteruptime_secrets" {
+	source = "../modules/secrets"
+	keys = [
+		"betteruptime/token",
+		"rivet/api_status/token",
+	]
+}
+
+resource "betteruptime_status_page" "status_page" {
+	company_name = var.better_uptime.company_name
+	company_url = var.better_uptime.company_url
+	timezone = "UTC"
+	subdomain = var.better_uptime.company_subdomain
+	design = "v2"
+	layout = "vertical"
+	theme = "light"
+}
+
+output "status_page_domain" {
+  description = "The domain of the Better Uptime status page"
+  value       = betteruptime_status_page.status_page.domain
+}
```

Comment:
should be in output.tf

Reply:
nit: don't use the tab-aligned equal signs

Reply:
`.domain` doesn't exist, were we using a diff version of the tf provider? unsure how this worked.